drivers/sensor/ti/tmp116/tmp116.c

```diff
@@ -255,7 +255,7 @@ static int tmp116_attr_set(const struct device *dev,
 		return -ENOTSUP;
 	}
 
-	switch (attr) {
+	switch ((int)attr) {
```

Comment:
```suggestion
	switch ((enum sensor_attribute_tmp_116)attr) {
```

Reply:
@jeppenodgaard, unfortunately I get compiler warnings with both `((enum sensor_attribute_tmp_116)attr)` and `((enum sensor_attribute)attr)`, casting to an `int` seems like the best option and is used [elsewhere...](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/ti/tmp108/tmp108.c#L174)